src/Wallabag/ImportBundle/Command/ImportCommand.php

```diff
@@ -34,10 +35,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
         // Turning off doctrine default logs queries for saving memory
         $em->getConnection()->getConfiguration()->setSQLLogger(null);
 
-        $user = $em->getRepository('WallabagUserBundle:User')->findOneById($input->getArgument('userId'));
+        if ($input->getOption('useUserId')) {
```

Comment:
You can do `findOneByUsername` by default and only have a `if` if we set a userId.